app/styles/app.scss

```diff
@@ -8,6 +8,9 @@
 @import 'materialize';
 @import 'ember-cli-materialize';
 
+/* font awesome variables */
+@import 'bower_components/font-awesome/scss/_variables.scss';
```

Comment:
This has two wrong things in it.

One: we don't have font-awesome in bower_components
Two: that's not how installed assets are to be included

However, because the order of things is important here, I symlinked the references scss file into the project and included the link. As far as I can tell, it should work just fine, but it needs a bit of checking as we didn't quite attempt this before.

I'll commit my fix shortly and see if the error goes away on Travis as well.